proto/injective/downtimedetector/v1beta1/query.proto

```
@@ -0,0 +1,42 @@
+syntax = "proto3";
+package injective.downtimedetector.v1beta1;
+
+import "gogoproto/gogo.proto";
```

Buf (1.57.2)

import "gogoproto/gogo.proto": file does not exist

(COMPILE)